content/reference/admin/private_locations/kubernetes_deployment/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Deploying a Control Plane on Kubernetes"
+description: "How to deploy a Gatling Control Plane on Kubernetes, to set up your Private Locations and run load generators in your own Kubernetes cluster"
+lead: "Run a Control Plane on Kubernetes, to set up your Private Locations and run load generators in your own Kubernetes network"
+weight: 22053
+---
+
+### Introduction
+
+A control plane, deployed on a kubernetes cluster with the needed roles will be able to deploy kubernetes private locations.
+
+To better understand what roles are needed, here's a quick overview of the mechanisms involved when deploying a private location on Kubernetes.
+
+First, the control plane is notified to deploy a number of instances for a given Kubernetes location.
+
+Then, a config map is created, given the namespace of the kubernetes location, with the starting script of a Gatling load generator.
+*This script is not embedded in our official images, to allow custom images*
+
+Finally, a batch job is initiated, with the number of instances configured on the simulation run, config map is mounted on each pod of the job.
+
+### Roles
+
+For each private location configured namespace, following roles are needed:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: gatling-control-plane-role
+  namespace: <namespace>
+rules:
+  - apiGroups: [""]
+    resources: ["configmaps"]
+    verbs: ["get", "create", "update", "delete"]
+  - apiGroups: ["batch"]
+    resources: ["jobs"]
+    verbs: ["create"]
+```
+
+### Example
+
+A deployment for a control plane on namespace `gatling`, using a previously configured config map `control-plane-config`
```

Comment:
Two things here : 
- As config-map has a key/value structure, i think we have to say that config map name is `control-plane-config`, and the key should be `control-plane.conf`
- More generaly, it's a bit abrupt to references this config map here, maybe we can put an example, what do you think ? Either the command to create config map from a file, or an example like : 

```yaml
apiVersion: v1
kind: ConfigMap
metadata:
  name: control-plane-config
  namespace: gatling
data:
  control-plane.conf: |
    control-plane {
      # Control plane token
      token = "cpt_eks_bastien_test_c7oze5djp3u14a5xqjanhc...."
      # Control plane description (optional)
      description = "Control plane on EKS"
      # Locations configurations
      locations = [
        {
          # Private location ID, must be prefixed by prl_, only lower alpha,
          # numeric, underscores and with a max length of 30 characters
          id = "prl_eks_bastien"
          # Private location description (optional)
          description = "Private Location on Kubernetes"
          # Private location type
          type = "kubernetes"
          # Namespace (optional, default based on kubernetes configuration)
          namespace = "gatling"
          # Certified image configuration
          image {
            type = certified
            java = 17 # Possible values : "8", "11" or 17"
          }
          # Custom image configuration
          # image {
          #   type = custom
          #   image = gatlingcorp/frontline-injector:17
          # }
          # Clean up finished jobs resources after given time (optional)
          ttl-after-finished = 10 minutes
          # Labels of initiated resources (optional)
          labels {
            LabelTest = BastienCP
            # ExampleKey = ExampleValue
          }
          # Annotations of initiated resources (optional)
          #annotations {
            # ExampleKey = ExampleValue
          #}
          # Environment variables of initiated pods (optional)
          #environment-variables {
            # ExampleKey = ExampleValue
          #}
          # Tolerations (optional)
          #tolerations = [
          #  {
          #    key = key1
          #    operator = Equal
              # Value is not needed when effect is Exists (optional)
          #    value = value1 
              # An empty effect matches all effects with key (optional)
          #    effect = NoSchedule
          #  }
          #]
        }
      ]
    }
---
```

Reply:
I prefer to refer to the documentation, and provide the command to create such a config map from a file:
- Easier to create that way
- Easier to maintain on documentation for us